appveyor.yml

```diff
@@ -46,7 +53,7 @@ before_build:
   - cmd: if "%configuration%"=="Release" call script\appveyor\cuda_install.cmd
 
 build_script:
-  - set PENGUINV_UNIT_TEST_RUN_COUNT=500
+  - set PENGUINV_UNIT_TEST_RUN_COUNT=5
```

Comment:
Do we need thia change? I think we needed this for testings only :)

Reply:
This was for testing obviously.